# HG changeset patch
# User Rik <rik@octave.org>
# Date 1484350897 28800
#      Fri Jan 13 15:41:37 2017 -0800
# Node ID a11950e51b7f1dbbe73dbf92585d374117e8c3e4
# Parent  d8d0e600cbc4ea5633a379255d2070fffd5d01c6
# Parent  1c98489534aae8f1d19b9b5ce36a614a8777c914
maint: merge stable to default.

diff --git a/etc/icons/octave.appdata.xml.in b/etc/icons/octave.appdata.xml.in
--- a/etc/icons/octave.appdata.xml.in
+++ b/etc/icons/octave.appdata.xml.in
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  Copyright (C) 2013-2016 Carnë Draug
+  Copyright (C) 2013-2017 Carnë Draug
 
   This file is part of Octave.
 
   Octave is free software; you can redistribute it and/or modify it
   under the terms of the GNU General Public License as published by
   the Free Software Foundation; either version 3 of the License, or
   (at your option) any later version.
 
@@ -13,21 +13,23 @@
   WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.
 
   You should have received a copy of the GNU General Public License
   along with Octave; see the file COPYING.  If not, see
   <http://www.gnu.org/licenses/>.
 -->
-<application>
-  <id type="desktop">%OCTAVE_DESKTOP_FILE%</id>
-  <licence>GPLv3+</licence>
+<component type="desktop">
+  <id>%OCTAVE_DESKTOP_FILE%</id>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0+</project_license>
   <name>GNU Octave</name>
   <summary>Interactive programming environment for numerical computations</summary>
+
   <description>
     <p>
       GNU Octave is a high-level interpreted language, primarily intended for
       numerical computations. It provides capabilities for the numerical
       solution of linear and nonlinear problems, and for performing other
       numerical experiments. It also provides extensive graphics capabilities
       for data visualization and manipulation. Octave is normally used through
       its interactive command line interface, but it can also be used to write
@@ -39,21 +41,25 @@
       problems, finding the roots of nonlinear equations, integrating ordinary
       functions, manipulating polynomials, and integrating ordinary
       differential and differential-algebraic equations. It is easily
       extensible and customizable via user-defined functions written in
       Octave's own language, or using dynamically loaded modules written in
       C++, C, Fortran, or other languages.
     </p>
   </description>
+
   <screenshots>
     <!--
-      From: http://people.freedesktop.org/~hughsient/appdata/
+      From: https://www.freedesktop.org/software/appstream/docs/
       All screenshots should have a 16:9 aspect ratio, and should have
       a width that is no smaller than 620px. Screenshots should be in PNG
       or JPEG format. PNG is the preferred format.
     -->
-    <screenshot type="default">http://www.gnu.org/software/octave/images/screenshot.png</screenshot>
+    <screenshot type="default">
+      <image>https://www.gnu.org/software/octave/img/screenshot.png</image>
+    </screenshot>
   </screenshots>
+
   <url type="homepage">http://www.octave.org</url>
-  <updatecontact>maintainers@octave.org</updatecontact>
+  <update_contact>maintainers@octave.org</update_contact>
   <project_group>GNU</project_group>
-</application>
+</component>
